src/pincode/authentication.ts

```diff
@@ -316,7 +317,11 @@ export async function setPincodeWithBiometry() {
 export async function getPincodeWithBiometry() {
   try {
     ValoraAnalytics.track(AuthenticationEvents.get_pincode_with_biometry_start)
-    const retrievedPin = await retrieveStoredItem(STORAGE_KEYS.PIN)
+    const retrievedPin = await retrieveStoredItem(STORAGE_KEYS.PIN, {
+      authenticationPrompt: {
+        title: i18n.t('unlockWithBiometryPrompt'),
+      },
+    })
```

Comment:
hum, so in theory this option is for both iOS and Android but i have not seen it appear on iOS....gona see if i can find where it's used.

Reply:
i think this would have been displayed on iOS if we allowed the device pincode as fallback, but since we don't then it's really only affecting android :) i've left a comment